lib/invoice.rb

```diff
@@ -0,0 +1,31 @@
+class Invoice
+
+    def initialize(items_list)
+        @items_list = items_list
+        puts print_table
+    end
+
+
+    def print_invoice (total_actual_price, total_sale_price)
```

Comment:
Split this into 2 methods called generate_total_actual_price and generate_total_sale_price. Further, use these methods where they are needed by a calling class.